src/scss/partials/_nav.scss

```diff
@@ -331,7 +331,8 @@ $nav-popover-padding          : spacing(x2);
 
     // Icon active state
     // Shown when the checkbox is checked
```

Comment:
Need to update inline comment so it’s relevant to both selectors

Reply:
Done.